dirent.go

```diff
@@ -57,6 +57,13 @@ func (de Dirent) IsRegular() bool { return de.modeType&os.ModeType == 0 }
 // set.
 func (de Dirent) IsSymlink() bool { return de.modeType&os.ModeSymlink != 0 }
 
+// IsDevice returns true if and only if the Dirent represents a device file.
+// Note that on some operating systems, more than one file mode
```

Comment:
Lines 61--64 do not apply to this particular method. The comment is there for IsSymlink() and IsDirectory only because in Windows both of those bits might be set.

Reply:
Updated as requested.